qrcode/console_scripts.py

```diff
@@ -114,15 +120,15 @@ def raise_error(msg: str) -> NoReturn:
 
     if opts.output:
         img = qr.make_image()
-        with open(opts.output, "wb") as out:
+        with Path(opts.output).open("wb") as out:
```

Comment:
This change doesn't make much sense to me. There's no need to waste cycles on converting the string to a Path, just to then call `open` on it, which will in turn call `open(self)`...

Reply:
True, its an automated fix by Ruff. https://docs.astral.sh/ruff/rules/builtin-open/ 

I don't mind it, but feel free to reverse if you think the performance issues outweight it. For me personally, I prefer the consistency of using either pathlib or open.